packages/lime-cc/Makefile

```diff
@@ -0,0 +1,60 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lime-cc
+PKG_VERSION:=0.0.2
+PKG_RELEASE:=2
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/lime-cc-client
+  SECTION:=lime
+  CATEGORY:=Lime
+  TITLE:=lime-cc-client
+  PKGARCH:=all
+  DEPENDS:=+bmx7-sms
+endef
+
+define Package/lime-cc-client/description
+	Checks for new config files and installs if sender is trusted
+endef
+
+define Build/Compile
+endef
+
+define Build/Configure
+endef
+
+define Package/lime-cc-client/install
+		$(INSTALL_DIR) $(1)/usr/bin
+		$(CP) files/lime-cc-client.sh $(1)/usr/bin/lime-cc-client
+		chmod +x $(1)/usr/bin/lime-cc-client
+
+		$(INSTALL_DIR) $(1)/etc/init.d
+		$(CP) files/lime-cc-client.procd $(1)/etc/init.d/lime-cc-client
+endef
+
+define Package/lime-cc-server
+  SECTION:=lime
+  CATEGORY:=Lime
+  TITLE:=lime-cc-server
+  PKGARCH:=all
+  DEPENDS:=+bmx7-sms
```

Comment:
It should also depend on bmx7 and lme-system.

Reply:
Also busybox cron (see smonit package)

Reply:
`bmx7-sms` depends on `bmx7` afaik, however I'll add `lime-system` thanks

Reply:
Even if bmx7-sms depends on bmx7 I would add bmx7 so the package do not depend on bmx7-sms package. Also it's a good practice to easy show what are the real dependencies.